jekyll/_cci2/(WIP) api-v2-pipelines-parameters-scopes.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: classic-docs
+title: "API v2 Pipelines, Parameters and Scopes"
```

Comment:
I feel like this information is more valuable overall in the context of pipelines and pipeline parameters; not specifically for using API v2. Maybe we should just put this as a section under [Advanced Config](https://circleci.com/docs/2.0/adv-config/#section=configuration) instead.

Reply:
@jama22 I can definitely add this to the Advanced Config section of our documentation and decouple this from API v2.